### PR TITLE
feat(bridge): Remove millis and micros from evaluation time frame

### LINKED ISSUE #7734: Trigger sequence: Milli and Micro seconds are not needed
### Task

When triggering an evaluation sequence, the timeframe select allows specifying a timeframe in hh : mm : ss : ms : µs - please see: 
 image

Please remove ~~ms : µs~~ from the date format since this is too specific.

### PR DESCRIPTION
**Pull request by heinzburgstaller**
## This PR
- Disable millis and micros for time input component
- Refactor setTimeframe component method
- Adapt cypress integration test

### Related Issues
Resolves #7734 

### How to test
1. Trigger sequence
2. Select evaluation
3. Time frame input only allows hours, minutes and seconds

### Screenshot
image

### CODE REVIEW

File: bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts [hunk 1]
```diff
@@ -199,18 +199,14 @@ export class KtbTriggerSequenceComponent implements OnInit, OnDestroy, AfterView
...
-    if (!this.isTimeframeEmpty(timeframe)) {
-      this.isValidTimeframe =
-        (timeframe.hours ?? 0) * 60 +
+    this.isValidTimeframe = this.isTimeframeEmpty(timeframe)
+      ? true
+      : (timeframe.hours ?? 0) * 60 +
           (timeframe.minutes ?? 0) +
           (timeframe.seconds ?? 0) / 60 +
           (timeframe.millis ?? 0) / 60_000 +
           (timeframe.micros ?? 0) / 60_000_000 >=
```
Comment (renepanzar) [inline bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts:208]: Is this still necessary?

Comment (heinzburgstaller) [reply bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts:208]: When it is still there, we could enable millis and micros again (if it is necessary).

Comment (renepanzar) [reply bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts:208]: I'd rather go for removing it, as we can also introduce it if needed later on. YAGNI principle applied, and less code is less maintenance :)

Comment (heinzburgstaller) [reply bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.ts]: Changed

**Review by renepanzar — APPROVED** [review]
LGTM!

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**